src/styles/arrow.scss

```diff
@@ -0,0 +1,46 @@
+.arrow {
+  position: relative;
+
+  &__minute {
+    display: flex;
+    justify-content: center;
+    position: absolute;
+    animation: rotate 3600s linear 1;
+
+    &:before {
+      content: "";
+      position: absolute;
+      width: 8px;
+      height: 180px;
+      border-radius: 6px;
+      background: saddlebrown;
+      z-index: 10;
+    }
+  }
+
+  &__second {
+    display: flex;
+    justify-content: center;
+    position: absolute;
+    animation: rotate 60s linear 10;
+
+    &:before {
+      content: "";
+      position: absolute;
+      width: 4px;
+      height: 250px;
+      border-radius: 6px;
+      background: #d3d3d3;
+      z-index: 11;
+    }
```

Comment:
styles for minute and second almost same. make mixin for them with arguments which different